### DISCUSSION TITLE
Scryers or Aldor choose quest problem.

### DISCUSSION THREAD

**Discussion by oldcat0414** [chromiecraft/chromiecraft#5276] [General]
My account doesn't have Allegiance to the Scryers(or Allegiance to the Aldor)
So I can't accept follow-up quests.

I have asked many friends, but none of them have this question

**Comment by 55Honey** [comment]
Hello @oldcat0414,
It's encouraged to open a ticket on our discord server, so we don't have to disclose your character details in a public channel like this.

GMs will need to know your character name to be able to look into your issue.
https://discord.gg/chromiecraft is where the community interacts for the most part. And also where players can open tickets.
